code/modules/surgery/organs/subtypes/standard_organs.dm

```diff
@@ -186,6 +187,26 @@
 
 	. = ..()
 
+/obj/item/organ/external/hand/necrotize(update_sprite)
+	. = ..()
+	update_hand_missing()
+
+/obj/item/organ/external/hand/mutate()
+	. = ..()
+	update_hand_missing()
+
+/obj/item/organ/external/hand/receive_damage(brute, burn, sharp, used_weapon, list/forbidden_limbs, ignore_resists, updating_health)
+	. = ..()
+	update_hand_missing()
+
+/obj/item/organ/external/hand/droplimb(clean, disintegrate, ignore_children, nodamage)
+	. = ..()
+	update_hand_missing()
+
+/obj/item/organ/external/hand/proc/update_hand_missing()
+	// we need to come back to this once the hand is actually removed/dead
+	addtimer(CALLBACK(owner, TYPE_PROC_REF(/mob/living/carbon/human, update_hands_hud), 0))
```

Comment:
Any reason why you're adding a timer and not just async invoking (or calling the proc itself)?